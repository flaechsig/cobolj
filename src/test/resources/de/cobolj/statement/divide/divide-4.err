Division eines Wertes auf mehrere Ziele (über By und Giving) ohne Speicherüberlauf