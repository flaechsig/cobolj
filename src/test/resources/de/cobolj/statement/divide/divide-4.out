KEIN SPEICHERUEBERLAUF
005
006
